src/wavesurfer.js

```diff
@@ -204,6 +208,8 @@ export default class WaveSurfer extends util.Observer {
         audioScriptProcessor: null,
         audioRate: 1,
         autoCenter: true,
+        autoCenteringRate: 5,
```

Comment:
I would call it `autoCenterRate`, to make it similar to the other autoCenter options.